examples/armada-manifest-v1.yaml

```diff
@@ -0,0 +1,101 @@
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: ucp-dependecy
+data:
+  name: helm-toolkit
+  release: helm-toolkit
+  namespace: null
+  values: {}
+  source:
+    type: git
+    location: git://github.com/openstack/openstack-helm
+    subpath: helm-toolkit
+    reference: master
+  dependecies: []
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: ucp-mariadb
+data:
+  name: mariadb
+  release: mariadb
+  namespace: undercloud
+  timeout: 3600
+  values: {}
+  source:
+    type: git
+    location: git://github.com/openstack/openstack-helm
+    subpath: mariadb
+    reference: master
+  dependecies:
+    - ucp-dependecy
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: ucp-memcache
+data:
+  name: memcache
+  release: memcache
+  namespace: undercloud
+  timeout: 100
+  values: {}
+  source:
+    type: git
+    location: git://github.com/openstack/openstack-helm
+    subpath: helm-toolkit
+    reference: master
+  dependecies:
+    - ucp-dependecy
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: ucp-keystone
+data:
+  name: keystone
+  release: keystone
+  namespace: undercloud
+  timeout: 100
+  values: {}
+  source:
+    type: git
+    location: git://github.com/openstack/openstack-helm
+    subpath: helm-toolkit
+    reference: master
+  dependecies:
+    - ucp-dependecy
+---
+schema: armada/ChartGroup/v1
+metadata:
+  schema: metadata/Document/v1
+  name: ucp-openstack-services
+data:
+  description: "Undercloud Identity Charts"
+  sequenced: True
+  chart_group:
+    - ucp-mariadb
+    - ucp-memcache
+---
+schema: armada/ChartGroup/v1
+metadata:
+  schema: metadata/Document/v1
+  name: ucp-openstack-group
+data:
+  description: "Undercloud Identity Charts"
+  sequenced: False
+  chart_group:
+    - ucp-keystone
+---
+schema: armada/Definition/v1
+metadata:
+  schema: metadata/Document/v1
+  name: ucp-armada-manifest
+data:
+  armada:
+    release_prefix: armada
+    charts:
```

Comment:
Probably want this key to be named `chartGroups`.

I would also include the mariadb + memcached group in this list, just to make the example more complete.